thumbor_mongodb/loader.py

```diff
@@ -7,24 +7,29 @@
 # Licensed under the MIT license:
 # http://www.opensource.org/licenses/mit-license
 
-from tornado.concurrent import return_future
-from pymongo import MongoClient
-from bson.objectid import ObjectId
 import gridfs
+from pymongo import MongoClient
+from tornado.concurrent import return_future
 
-def __conn__(self):
-	connection = MongoClient(
-		self.config.MONGO_STORAGE_SERVER_HOST,
-		self.config.MONGO_STORAGE_SERVER_PORT,
-		readPreference=self.config.MONGO_STORAGE_READ_PREFERENCE
-	)
 
-	db = connection[self.config.MONGO_STORAGE_SERVER_DB]
-	return db
+def __conn__(self):
+    connection = MongoClient(
+        self.config.MONGO_STORAGE_SERVER_HOST,
+        self.config.MONGO_STORAGE_SERVER_PORT,
+        readPreference='primaryPreferred'
```

Comment:
please leave this for config, for example I have environment where it to reads from slave and master is only for write